docs/serviceDiagram.md

```diff
@@ -4,30 +4,22 @@ Overview how the different services interact with each other.
 
 ## Flow
 
-Data flow for a GraphQL request which also publishes subscription data.
+Data flow for a GraphQL mutation by Client 1 which also publishes subscription data to Client 2.
 
 ```mermaid
 flowchart TD
-  Client
-  Server[Websocket/Subscription server]
-  RedisStream(Redis Stream)
-  RedisResultPubSub(Redis PubSub\ngqlExRes)
-  RedisSubscriptionPubSub(Redis PubSub\nTeam.teamId)
-  subgraph "per executor {executorChannel}"
-    GQLExecutor[GraphQL Executor]
-    RedisServerPubSub(Redis PubSub\nexecutorChannel)
-    RedisServerPubSub --> GQLExecutor
-  end
-
-  click Server "../packages/server/server.ts"
-
-  Client <---->|Websocket| Server
-  Server -->|gqlStream| RedisStream -->|gqlConsumerGroup| GQLExecutor
-  GQLExecutor --> RedisResultPubSub
-  GQLExecutor --> |"{executorChannel}"| RedisSubscriptionPubSub
-  RedisSubscriptionPubSub --> |"{executorChannel}"|Server
-  Server --> RedisServerPubSub
-  RedisResultPubSub --> Server
+  Client1(Client 1)
+  Client2(Client 2)
+  Server1(Server 1 - GraphQL Resolver)
+  Server2(Server 2 - GraphQL Resolver)
+  RedisSubscriptionPubSub(Redis PubSub)
+
+  click Server1 "../packages/server/server.ts"
```

Comment:
This link isn't working for me

Reply:
I think this is a preview issue. I didn't change the link.

Reply:
It is, I just checked